dash-renderer/VERSION.txt

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
Would\
(1) include this file in `.gitignore` for the `dev` branch -- but not in `master`
(2) generate it from the `package.json`